### ISSUE TITLE
Error in alignment parsing

### ISSUE THREAD

**Issue by roblanf** [brettc/partitionfinder#73]
Here's an alignment:

```
3 2
a AT
b AT
c AT
d AC
```

When I try and run it in PF, I get a misleading error:

```
Line 5: Expected a blank line between blocks
```

@brettc, can we fix this? The correct error here is that the number of species in the header is wrong. We used to detect this fine. But now we don't.

Also, can we add a test for this so we don't do it again.

R

**Comment by brettc** [comment]
Oops. Will sort it.

**Comment by brettc** [comment]
Fixed in 33d919c0ae4dac27c01ed9fc80f51bc08681fae9
